### DISCUSSION TITLE
Keymap art / cheat sheet for desk mat

### DISCUSSION THREAD

**Discussion by 0x241F31** [helix-editor/helix#5599] [General]
![image](https://user-images.githubusercontent.com/17229619/213528366-3358fa71-2208-4466-aa3a-02f92bc9beb5.png)
Is there something like this so I can order to print on desk pad (or keycaps) and remember?

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/wiki/Keymap-brainstorm
Has a picture, but it's a bit out-of-date.

**Reply by peterjc** [reply]
That wiki page no longer exists - there is a snapshot just over a year ago: https://web.archive.org/web/20240111090600/https://github.com/helix-editor/helix/wiki/Keymap-brainstorm with this image:

![Image](https://web.archive.org/web/20240111090600im_/https://user-images.githubusercontent.com/91177333/192541110-19a94459-9467-41f6-bbd8-56d02e99ba32.png)

Given another two years have past, it is likely even more out of date

**Comment by sshaikh** [comment]
I found the OG vi cheatsheet a godsend, so this would be of value. Perhaps there's a way to have one auto-generated from config?

**Comment by netshade** [comment]
I took a crack at making one for myself. Was going to give myself a few days to double check all the bindings, then print a desktop mat of this. Happy for any suggestions here. ( PNG attached is quantized and crushed, so would not recommend printing from this image )

<img width="9450" height="4650" alt="pngout" src="https://github.com/user-attachments/assets/849912c3-cc36-489e-87b5-6731c8591851" />
